metadata/slack.json

```diff
@@ -436,11 +437,12 @@
                         ],
                         "returnValue": {
                             "name": "string",
-                            "displayName": "Channel Id"
+                            "displayName": "Channel ID"
                         }
                     }
                 ]
             }
-        ]
+        ],
+        "listener": {}
     }
 }
```

Comment:
Shall we add client display name as "Slack Client" instead of "Base Client" ? I think it will be more user friendly. And shall we rename the FileInfo display name as "File Information"  if it's convenient?

Reply:
Sure.